style.css

```diff
@@ -0,0 +1,786 @@
+    background-repeat: no-repeat;
+    filter: sepia(100%) brightness(2);
+    opacity: 0.1;
+  }
+
+  .nav-list {
+    padding-left: 0;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    list-style: none;
+    border: 1px solid #272a312f;
+    background-color: white;
+    margin-top: 0;
+  }
+
+  .nav-list img {
+    width: 50px;
+  }
+
+  .menu-links {
+    padding-right: 20px;
+    text-decoration: none;
+    color: black;
+  }
+
+  .about {
+    padding-right: 20px;
+    text-decoration: none;
+    color: #ec5242;
+  }
+
+  .menu-links-project {
+    text-decoration: none;
+    border: 3px solid #ec5242;
+    height: 30px;
+    margin-right: 20px;
+    width: 100px;
+    text-align: center;
+  }
+
+  .menu-links-project a {
+    color: #ec5242;
+    font-family: "LatoWebThin", sans-serif;
+    vertical-align: baseline;
+    font-size: medium;
+    font-weight: bolder;
+  }
+
+  .list-container {
+    display: flex;
+    padding-right: 15%;
+    gap: 35px;
+  }
+
+  .list-container li {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .nav-list a {
+    padding-left: 15%;
+  }
+
+  .container-p {
+    display: none;
+  }
+
+  .header-p-desktop {
+    width: 50%;
+    text-align: justify;
+    padding-left: 10px;
+    background-color: #e8e8e9;
+    margin-left: 15%;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    padding-right: 10px;
+    border: 3px solid white;
+  }
+
+  .header-day {
+    display: none;
+  }
+
+  .header-place {
+    display: none;
+  }
+
+  .header-day-desktop {
+    padding-left: 15%;
+    font-family: "LatoWebBlack", sans-serif;
+  }
+
+  .header-place-desktop {
+    padding-left: 15%;
+  }
+
+  .program-section {
+    display: flex;
+    flex-direction: column;
+    background-color: #272a31;
+    color: white;
+  }
+
+  .grid-container {
+    display: none;
+  }
+
+  .grid-container-desktop {
+    display: flex;
+    flex-direction: row;
+    background-image: url(sources/mobile-version/images/background-program4.jpg);
+    background-size: contain;
+  }
+
+  .card {
+    text-align: center;
+    width: 300px;
+    height: 306px;
+    margin-left: 20px;
+    margin-right: 20px;
+    background-color: rgba(60, 64, 75, 0.5);
+  }
+
+  .card h2 {
+    color: #ec5242;
+  }
+
+  .card h3 {
+    font-size: medium;
+  }
+
+  .button-container {
+    display: none;
+  }
+
+  .link-desktop {
+    margin-top: 70px;
+    color: white;
+    font-family: "LatoWebBold", sans-serif;
+    margin-bottom: 70px;
+    text-align-last: center;
+  }
+
+  .program {
+    text-align-last: center;
+    margin-top: 40px;
+  }
+
+  .tiny-line {
+    width: 2rem;
+    height: 3px;
+    background-color: #ec5242;
+    margin: auto;
+    margin-bottom: 2rem;
+  }
+
+  .skrillex {
+    width: 150px;
+    height: 150px;
+    margin-right: 15px;
+  }
+
+  .skrillex img {
+    padding-top: 20px;
+    width: 100%;
+    border-radius: 50%;
+  }
```

Comment:
- [ ] I also noticed you seem to be missing the gray pattern background on each speaker image as shown below so I highly advise you to add it in order to stay consistent with the original design.
- Behance design:
 ![image](https://github.com/SaveryIV/First-Capstone-Project/assets/104968236/dbac4b6c-4f2e-4d11-a21e-26fb31cdabfc)
- Your design:
 ![image](https://github.com/SaveryIV/First-Capstone-Project/assets/104968236/14638d2c-9129-43d0-97f7-346d72ccbebb)